### ISSUE TITLE
A more optimal way to do __eq__

### ISSUE THREAD

**Issue by YehudaCorsia** [wolverdude/GenSON#29]
Hey, i saw that you search for more optimal way to do `__eq__` in `SchemaNode`
and i am also saw that you use not optimal way in `SchemaBuilder`  

and also in `SchemaNode` you do 
`return self.to_schema() == other.to_schema()`
and `to_schema()` is big function that might take a lot of time.

i am think have a better solution that you can use:
```python
def __eq__(self, other):
    return (isinstance(other, self.__class__) and
           self.__dict__ == other.__dict__)

**Comment by YehudaCorsia** [comment]
I just created a pull request with the new code...

**Comment by wolverdude** [comment]
This should be updated in the next release.

**Comment by wolverdude** [comment]
Fixed in `v1.2.0`
